src/codegen.cpp

```diff
@@ -7159,10 +7159,11 @@ extern "C" void jl_init_codegen(void)
     initializeAnalysis(Registry);
 #endif
 
-    InitializeNativeTarget();
-    InitializeNativeTargetAsmPrinter();
-    InitializeNativeTargetAsmParser();
-    InitializeNativeTargetDisassembler();
+    InitializeAllTargets();
```

Comment:
Do we really need to initialize **ALL** targets? Distributions usually build LLVM with all targets enabled and this seems like a huge waste.